docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -134,6 +134,7 @@ Standard TensorFlow\* operations:
 | BatchToSpaceND | No |
 | BiasAdd | No |
 | Bucketize | CPU only |
+| Broadcast_to| No |
```

Comment:
Change to `BroadcastTo`. We use TF op low-level names in this table (not python API names)

Reply:
done